src/app/story/directives/podcast.component.ts

```diff
@@ -1,96 +1,86 @@
 import { Component, OnDestroy } from '@angular/core';
```

Comment:
So close! This should also 
 - [ ] default the author name and email to the series author name and email
 - [ ] and show the enclosure url (readonly)